tests/unit/Pelagos/Entity/DifTest.php

```diff
@@ -0,0 +1,316 @@
+<?php
+
+namespace Pelagos\Entity;
+
+use PHPUnit\Framework\TestCase;
+
+class DifTest extends TestCase
+{
+
+    /**
+     * @var DIF
+     */
+    protected $dif;
+
+    /**
+     * @var ResearchGroup
+     */
+    protected $mockResearchGroup;
+
+    /**
+     * Setup for all the test cases in National Data center entity test.
```

Comment:
I guess this test came from National Data center!? ;-)

Reply:
Haha.. not the entire test  :P I think just the setup